docs/en/contributor/Building-Alluxio-From-Source.md

```diff
@@ -94,12 +95,11 @@ Once Alluxio is built, you can validate and start it with:
 ```console
 $ # Alluxio uses ./underFSStorage for under file system storage by default
 $ mkdir ./underFSStorage
-$ ./bin/alluxio validateEnv local
```

Comment:
why this is removed?

Reply:
When cloning the repo it produces `1 failures 4 warnings 7 skipped` as output, but the failure does not actually impede the process of building it from source.